py/circuitpy_defns.mk

```diff
@@ -390,6 +390,7 @@ SRC_COMMON_HAL_ALL = \
 	alarm/time/TimeAlarm.c \
 	alarm/touch/TouchAlarm.c \
 	analogio/AnalogIn.c \
+	analogio/AnalogFastIn.c \
```

Comment:
The reason the builds are failing is this functionality only exists on one port. You will have to include #ifdefs to turn it off on boards that it is not implemented on (or may not be due to space concerns). You will see many places with `#ifdef CIRCUITPY_FEATURE`. For example CIRCUITPY_ZLIB (something I did). If you search you will see the places you may need a `#ifdef CIRCUITPY_ANALOGFASTIN`

Then you will have to define it as off by default and in `ports/raspberrypi/mpconfigport.mk` have a line like `CIRCUITPY_ANALOGFASTIN = 1` to turn it on for that port.